docs/quantize.qmd

```diff
@@ -0,0 +1,47 @@
+---
+title: "Quantization with torchao"
+back-to-top-navigation: true
+toc: true
+toc-expand: 2
+toc-depth: 4
+---
+
+Quantization is a technique to lower the memory footprint of your model, potentially at the cost of accuracy or model performance. We support quantizing your model using the [torchao](https://github.com/pytorch/ao) library. Quantization is supported for both post-training quantization (PTQ) and quantization-aware training (QAT).
+
```

Comment:
Do we need to add a note that, we don't support quantize to gguf/gptq/exl2 at the moment? That may be people's first thought when they see `quantize` cli.